src/ducks/splits.js

```diff
@@ -2,19 +2,44 @@ import { Split } from 'seven-ten';
 import apiClient from 'panoptes-client/lib/api-client.js';
 
 const FETCH_SPLIT = 'FETCH_SPLIT';
+const FETCH_SPLIT_SUCCESS = 'FETCH_SPLIT_SUCCESS';
+const FETCH_SPLIT_ERROR = 'FETCH_SPLIT_ERROR';
 const CLEAR_SPLITS = 'CLEAR_SPLITS';
 const TOGGLE_OVERRIDE = 'TOGGLE_OVERRIDE';
 
+
+const SPLIT_STATUS = {
+  IDLE: 'split_status_idle',
+  FETCHING: 'split_status_fetching',
+  READY: 'split_status_ready',
+  ERROR: 'split_status_error',
+};
+
 const initialState = {
   adminOverride: false,
-  data: null
+  variant: 'individual',
```

Comment:
The `'individual'` and `'collaborative'` strings can probably be set as CONSTANT_VALUES if we're using them in a lot of places. `export const VARIANT_TYPES = { INDIVIDUAL: 'individual', COLLABORATIVE: 'collaborative' }` ?